model/resource/service_experimental.yaml

```diff
@@ -35,3 +35,15 @@ groups:
           (services aiming for reproducible UUIDs may also use Version 5, see RFC 4122
           for more recommendations).
         examples: ["my-k8s-pod-deployment-1", "627cc493-f310-47de-96bd-71410b7dec09"]
+      - id: type
+        type: string
+        brief: >
+          The service type in reverse domain name notation.
+        note: >
+          The `service.type` identifies the product that is deployed as the service.
+          The same product may be simultaneously deployed multiple times on the
+          same observable infrastructure. In this case each of these deployments
+          will typically have a distinct `service.name` to help identify the
+          logical role of the particular deployment, however their
+          `service.type` will be the same and will help identify the deployed product.
+        examples: ["io.opentelemetry.collector", "io.redis", "org.apache.kafka"]
```

Comment:
Does this need to be unique in some way?  The reverse-domain-name notations seems to imply it.

Should the service own the rights to the domain name it uses?

Reply:
The reverse-domain-name notation is our recommendation that helps prevent value conflicts without centralized coordination, assuming everyone plays by the rules.

I don't think we can enforce this and I don't think we want to verify the owning rights.

We follow this practice elsewhere (e.g. company-specific attribute names) and similarly don't attempt to enforce/verify it.

Reply:
Can you point me where that occurs?  I think that's (somewhat new) to semconv itself.

Reply:
I was referring to the [Recommendations for Application Developers](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/attribute-naming.md#recommendations-for-application-developers).

Reply:
@jsuereth is it clear? Can I resolve this?